packages/client/hooks/useServiceWorkerUpdater.ts

```diff
@@ -16,12 +16,13 @@ const useServiceWorkerUpdater = () => {
       }
       atmosphere.eventEmitter.emit('addSnackbar', {
         key: 'newVersion',
-        autoDismiss: 0,
-        message: 'A new version of Parabol is available',
+        autoDismiss: 5,
+        message: 'A new version of Parabol is available 🎉',
         action: {
-          label: 'Refresh to upgrade',
+          label: `See what's changed`,
           callback: () => {
-            window.location.reload()
+            const url = 'https://github.com/ParabolInc/parabol/blob/master/CHANGELOG.md#parabol-change-log'
```

Comment:
Maybe including the changelog will make users more aware that we're open source and attract devs 🤔